### DISCUSSION TITLE
Description for upscaling models

### DISCUSSION THREAD

**Discussion by josip01** [upscayl/upscayl#545] [Ideas]
Is it possible to add a short description for the upscaling models in the software that would show what the model is best suited or created to be used for? Maybe when You select the model the description would appear underneath. Right now I basically have to test each model to see which one works best for the selected image. 

If there would be a description like the description on github (4x_NMKD-Siax_200k - NMKD Siax - Universal upscaler for clean and slightly compressed images (JPEG quality 75 or better)) it may speed up the process of selecting the model.

Thank You for Your time.

**Comment by NayamAmarshe** [comment]
Yeah, I was thinking about that too. Maybe we'll add some comparison pictures in v3

**Reply by josip01** [reply]
Actually, I was thinking more like the model ID being  "modelname - best used for very compressed images" or something to that extent, but example comparison images would be great.
